pkg/core/pipeline/conditions.go

```diff
@@ -10,8 +10,10 @@ import (
 // RunConditions run every conditions for a given configuration config.
 func (p *Pipeline) RunConditions() (globalResult bool, err error) {
 
-	logrus.Infof("\n\n%s:\n", strings.ToTitle("conditions"))
-	logrus.Infof("%s\n", strings.Repeat("=", len("conditions")+1))
+	if len(p.Conditions) > 0 {
```

Comment:
Different approach but what do you think to exit early if nothing need to be done like 

```
if len(p.Conditions) == 0 {
    // Return early as no condition to run
    logrus.Debugln("no condition defined")
    return nil
}
```

Reply:
Oooh I like it! Let me take a look

Reply:
@olblak `logrus.Debugln` or `logrus.Infof` so the user knows why there isn't any output for the stage?

Reply:
I went with `logrus.Debugln` as you suggested.

Reply:
I think debug is better as it doesn't make sense to me tell the user that he didn't specify a condition or a source (or a target)